file/file_prefetch_buffer.h

```diff
@@ -379,12 +389,21 @@ class FilePrefetchBuffer {
   void PrefetchAsyncCallback(FSReadRequest& req, void* cb_arg);
 
   void TEST_GetBufferOffsetandSize(
-      std::vector<std::pair<uint64_t, size_t>>& buffer_info) {
+      std::vector<std::tuple<uint64_t, size_t, bool>>& buffer_info) {
```

Comment:
I added this third field to validate async prefetching, since otherwise I cannot distinguish between `async_req_len` and `CurrentSize()`